_posts/2018-08-08-cockpit-175.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 175
+author: mvo
+date: 2018-08-08
+tags: cockpit linux
+slug: cockpit-175
+category: release
+summary: Network bound disk encryption
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 175.
+
+### Network bound disk encryption
+
+It is now possible to manage the key slots of LUKSv1 encrypted
+devices. In addition to traditional passphrases, key slots can also be
+bound to a [Tang](https://github.com/latchset/tang) server via
+[Clevis](https://github.com/latchset/clevis).
+
+<iframe width="960" height="720" src="https://www.youtube.com/embed/GAPJVN6EEz4?rel=0" frameborder="0" allowfullscreen></iframe>
```

Comment:
If you'd like, I can clean up the audio on the video. Just give me the original file (somehow) and I'll process it.

Reply:
It#s just for the release notes, I think it's fine.  I should ask for a proper mike.